### ISSUE TITLE
Pressing Esc to return to lighttable only works when canvas has focus

### ISSUE THREAD

**Issue by qosch** [hanatos/vkdt#40]
Steps to reproduce:
- open an image in darkroom
- click somewhere in the right area with histogram and slider, press Esc without clicking anywhere else before
- nothing happens
- click somewhere in the image canvas, press Esc without clicking anywhere else before
- UI returns to lighttable

**Comment by hanatos** [comment]
yes that was by design once because some controls might be active and want to do something else with the esc/circle input. will need to carefully go through the ui controls and see if that is still the case. i agree that using a global esc to exit would be better.
